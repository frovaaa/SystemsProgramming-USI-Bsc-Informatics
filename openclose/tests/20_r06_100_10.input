bppbbb33b3bpqbp0ppbpqdbb0ppbdppbppqqdqppbbpqbb4ddpbbbdpbdbppbbbdddqqdqddqddqqqqqddqqqddddddqqd
pbbdd3pqbdbdppbbddbdpbdqqqq7pqbpbdbpqppbbddqbpqpqpqdpp4bdpq6ppqbdbpppqpbbpbdqpqpbdqddqqqdqqqqdqd
bdbpqd77bdbbpqdppqppppqpqqqqbppppqbdpbbdb8b2bbpqdpqbdpbbdbbppp9bd8bdbbbddpqdqqqdddqddddqqqqdqd
pq1b0bdppqqdpppbdbpbb7bpbbppqpbbppqqbpqpqdpqbbdppqpbpbpbp6bdpbpqpqdppqqqqdqdqdqqdddqqddqdddqdqqq
pp686pqbdbbbpbbdbbpqbpqbppbbpbbppbdqbb2pbbdppqbpqdqbbppqbbbbdpqdbpqdddqdddqddqddqddqqdddddqdddqq
5bdpppqqpppqbp7pbdp0pq0ppppbp07pqqdqqpqpqqbdqbdpq3pqb7pqpbpqppqppbdpqqpbbd9bddqqqdqdqqqdqqq
p3pp0qbdqqbdbpbbpbbppqbdqddqbpbdppbbppbpppbbpbbdpq9pbpbdpbb3bdbdppqqbdddqqdqdqddqqqdqqddqqqdddqd
8pbdqpppqqqbpqdbdbpqbdbdpbbbpppbpqbppqppbpbpqpbdbpbpqbbpqdbppbbdbbpqpbdqdddqqdddqdqdqdqqqddqqqdddqd
bp7qdbbpqdpq2dpqpqpbppqpbb0bpbdpqqddbpbppqbb0ppbbdbpbbpbpp6pqpqbdpqqqdb9pqdqddqddqqddqdqddqqdq
pbdbbpbpbbpqdpqb6bppbdpbdbpqdqpbpbbbdbpqddpqdq8pqpbbb4ddbddbppbdqqpbbddqpppqqqp8qdqdqqqdddqdqddq
