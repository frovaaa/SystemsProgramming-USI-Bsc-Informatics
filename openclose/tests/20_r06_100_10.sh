#!/bin/bash
exec "$1" "bdpq" < tests/20_r06_100_10.input
