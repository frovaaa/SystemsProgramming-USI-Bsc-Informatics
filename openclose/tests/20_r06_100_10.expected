bppbbb33b3bbp0ppbb0pppppbbbb4ddddqqqqqddqqqddddddqqd
p3q7bpbppp46qqqdqd
77bpbppppbb8b2bpbbbppp98qqqdddqddddqqqqdqd
1b0dpppbpbb7bpbbppbbbppbpbpbp6qdqdqdqqdddqqddqdddqdqqq
pp686bbbpbbbbbppbbpbbpbb2ddqddqddqqdddddqdddqq
5pppbp7pp00ppppbp07qdqqqq3b7pbpppb9dqqqdqdqqqdqqq
p3pp0qqqbpbbbpppbbppbpppbbpb9pbppbb3ddqqdqdqddqqqdqqddqqqdddqd
8
bp7qdb2dpbppbb0dbpbpbb0ppbbpbbpbpp6qqdb9dqddqddqqddqdqddqqdq
pbbpbpbb6bpppb8pbbb4dddbp8qdqdqqqdddqdqddq
